liquidity-module/index.md

```diff
@@ -359,10 +420,14 @@ Make sure to replace `cosmosxxxxx` with your address.
 You can also withdraw tokens from the pool you have created.
 
 ```bash
-gaiad tx liquidity withdraw 1 100pool-id --from username --chain-id cosmoshub-testnet --gas-prices "0.025stake" --node https://rpc.testnet.cosmos.network:443
+gaiad tx liquidity withdraw 1 100pool-id --from username --chain-id cosmoshub-testnet --gas-prices "0.025uphoton" --node https://rpc.testnet.cosmos.network:443
 ```
 
-- Replace `pool-id` with your pool id that you can see on [https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/cosmosxxx](https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/cosmosxxx)
+Make sure to replace the first `1` with your `pool id` obtained in previous step.
+
+- Replace `pool-id` with the id that you can see on [https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/cosmosxxx](https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/cosmosxxx)
+
+You will see something similar to `poolFE384B6C9AE769A67754EAD275832A5A6DAA6E2C769A1E1D281E596DD47AA36D`
```

Comment:
```suggestion
    You see a pool id that is similar to `poolFE384B6C9AE769A67754EAD275832A5A6DAA6E2C769A1E1D281E596DD47AA36D`.
```

Reply:
let's indent (you can tab or use four spaces like I did here) so this second line is nested under the associated list item :)